airflow/www/templates/airflow/dags.html

```diff
@@ -215,13 +215,13 @@ <h2>{{ page_title }}</h2>
                   title="Status of all previous DAG runs.">info</span>
           </th>
           <th>Schedule</th>
-          <th style="width:180px;">{{ sortable_column("Last Run", "last_dagrun") }}
+          <th style="width:180px;">{{ sortable_column("Last Data Interval", "last_dagrun_data_interval") }}
             <span class="material-icons text-muted js-tooltip" aria-hidden="true"
-                  title="Date/Time of the latest Dag Run.">info</span>
+                  title="Starting Date/Time interval of the latest Dag Run.">info</span>
           </th>
-          <th style="width:180px;">{{ sortable_column("Next Run", "next_dagrun") }}
+          <th style="width:180px;">{{ sortable_column("Next Data Interval", "next_dagrun_data_interval") }}
```

Comment:
I don't think we want to change the values, just the label, right?

Reply:
Do you mean that `Last Run` is the label and the `last_dagrun` is the value? 
I went ahead and made this modification as it seemed more appropriate to change both rather than just one of them as they are related. Let me keep the changes to label only